.github/workflows/checkAddonUrlAndHash.js

```diff
@@ -0,0 +1,73 @@
+const glob = require('glob');
+const crypto = require('crypto');
+const { v4: uuidv4 } = require('uuid');
+
+module.exports = ({core}, globPattern) => {
+  const fs = require('fs');
+  const { exec } = require('child_process');
+  const files = glob.globSync(globPattern);
+  files.forEach(file => {
```

Comment:
Can these be renamed to something slightly more descriptive please?
```suggestion
  const metadataFiles = glob.globSync(globPattern);
  metadataFiles.forEach(metadataFile => {
```